lib/src/main/java/ca/jbrains/pos/PointOfSale.java

```diff
@@ -7,20 +7,20 @@
 
 public class PointOfSale {
     public static void main(String[] args) {
-        parseInput(new InputStreamReader(System.in)).forEachOrdered(
-                line -> displayToConsole(displaySellOneItem(line, (ignored) -> "::a hardcoded response for every barcode::"))
+        readBarcodeFrom(new InputStreamReader(System.in)).forEachOrdered(
```

Comment:
I think the intention is readingBarcode from multiple input sources, so readBarcodeFrom provide the intention better

Reply:
Yes.

Even so, this function reads multiple barcodes, and not only one, therefore `readBarcodesFrom()`.